rg_tool.py

```diff
@@ -17,7 +17,7 @@
 except:
     pass
 
-TARGETS = ["odroid-go", "mrgc-g32", "qtpy-gamer", "retro-esp32"]
+TARGETS = ["odroid-go", "mrgc-g32", "qtpy-gamer", "retro-esp32", "esplay-s3"]
```

Comment:
This changes conflicts with a change I've just made, line should be:

`TARGETS = ["odroid-go"] # We just need to specify the default, the others are discovered below`

Sorry about that!